doc/source/data/creating-datasets.rst

```diff
@@ -304,6 +304,26 @@ are supported for each of these storage systems.
     :start-after: __read_parquet_az_begin__
     :end-before: __read_parquet_az_end__
 
+.. tabbed:: SQL Databases
```

Comment:
SQL Databases is not remote storage. When people talk about remote storage, nowadays they usually refer to remote / cloud-based file system (storage system), such as HDFS, S3, GCS, etc.

Let's make a separate section for "From SQL Databases".